cli/util/audit-bin.go

```diff
@@ -147,9 +171,18 @@ func dropBlock(ctx *cli.Context, p *pool.Pool, signer user.Signer, containerID c
 	if debug && err == nil {
 		fmt.Fprintf(ctx.App.Writer, "block duplicate %s for %d is removed (%s kept)\n", objID, prevH, curOID)
 	}
+	if err != nil {
+		err = fmt.Errorf("failed to remove block duplicate %s / %s (%d): %w", objID, curOID, prevH, err)
+	}
```

Comment:
It's a part of the previous commit.

Reply:
Yes and no, it's just a part of the error message that gets moved, the restore part _also_ has it before the last patch, but there it's deleted because there is enough data in messages from the function already (this one is duplicating). But can be restructured if you wish.